pages/api/Schema/Appeals/datahandler.ts

```diff
@@ -0,0 +1,66 @@
+import { Contact } from '../../../../graphql/types.generated';
+import { Appeal } from '../../graphql-rest.page.generated';
+
+export interface AppealsResponse {
+  id: string;
+  type: string;
+  attributes: {
+    amount: string;
+    created_at: string;
+    currencies: string[];
+    description: string;
+    end_date: string;
+    name: string;
+    pledges_amount_not_received_not_processed: number;
+    pledges_amount_processed: number;
+    pledges_amount_received_not_processed: number;
+    pledges_amount_total: number;
+    total_currency: string;
+    updated_at: string;
+    updated_in_db_at: string;
+  };
+  relationships: {
+    account_list: {
+      data: {
+        id: string;
+        type: string;
+      };
+    };
+    contacts: {
+      data: Contact[];
```

Comment:
I assume this isn't the right type since `Contact` is from GraphQL and this is typing the REST response.

You don't seem to be using any of the relationships so idk if it's worth typing.

Reply:
Yeah I wasn't sure what type to use but for the individual appeal pages there is a list of contacts. I didn't check if that was that same array or if its coming from somewhere else but either way I just removed all the extra stuff.